google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineRuntimeInstance.java

```diff
@@ -70,8 +72,11 @@ public void deploy(@NotNull final DeploymentTask<AppEngineDeploymentConfiguratio
 
     final AppEngineDeployAction deployAction;
     AppEngineDeploymentConfiguration deploymentConfig = task.getConfiguration();
-    File deploymentSource = deploymentConfig.isUserSpecifiedArtifact() ?
-        new File(deploymentConfig.getUserSpecifiedArtifactPath()) : task.getSource().getFile();
```

Comment:
hadn't we setup task.getSource().getFile() to return the user specified artifact so that we didn't have to special case this?